google-postgresql.yml

```diff
@@ -124,6 +124,33 @@ provision:
     constraints:
       maximum: 7
       minimum: 0
+  - field_name: highly_available
+    type: boolean
```

Comment:
**suggestion (blocking): expose `availability_type` as `string` or `enum`**

What are the benefits of abstracting away from Terraform's [availability_type](https://registry.terraform.io/providers/hashicorp/google/latest/docs#availability_type) enum ("REGIONAL"|"ZONAL")?

The downsides I see are:
- The additional indirection, abstraction an maintenance of wrapping the property.
- A possible (although unlikely) limitation if GCP comes up with a third possible value.

Reply:
Andrea and I discussed the pros and cons of this implementation.
The biggest pro he shared, in my opinion, is that MySQL is implemented in the exact same way.
I'm ok going with this implementation for consistency. We can decide to refactor both implementations in the future if needed (although it is very unlikely we will do so because it would imply forcing customer to change their plans and instances).